metadata.yaml

```
@@ -40,7 +40,7 @@ resources:
       OCI image for alertmanager. This charms makes the following assumptions about the image:
       - location of executable "alertmanager" is in the path
       - has `update-ca-certificates`
-    upstream-source: ubuntu/alertmanager:0.25.0-22.04_stable
+    upstream-source: ubuntu/alertmanager:0-22.04
```

At first glance I thought the version tag is wrong because of the odd zero :)

For workloads that are in the zero major version, I think we should go down to minor version.
We do not want big breaking changes going unnoticed.

```suggestion
    upstream-source: ubuntu/alertmanager:0.25-22.04
```

---

Big breaking changes don't happen on minor versions; tracking `0` means that the charm will use `0.25`, `0.26`, etc; if `1.0` comes out, it won't be a `0.x` version, so it won't be tracked by this tag.

We discussed this already multiple times, but we agreed that jumping between minor versions is fine :)